README.md

```diff
@@ -19,7 +19,7 @@ Built with top-notch technologies. Django, PostgreSQL, ElasticSearch, GraphQL an
 
 ## 💾 Installation and requirements
 
-Saleor requires Python 3.4+, Node.js 8.0+, PostgreSQL and OS-specific dependency tools.
+Saleor requires Python 3.5+, Node.js 8.0+, PostgreSQL and OS-specific dependency tools.
```

Comment:
It should probably also say Node.js 10+

Reply:
Also, Python and Node versions get listed in other places, like installation docs for linux